crates/net/downloaders/src/file_client.rs

```diff
@@ -127,12 +127,14 @@ impl FileClient {
     }
 
     /// Use the provided bodies as the file client's block body buffer.
+    #[allow(dead_code)]
     pub(crate) fn with_bodies(mut self, bodies: HashMap<BlockHash, BlockBody>) -> Self {
```

Comment:
if we were to use these methods in tests only, they should have been`#[cfg(test)]`, but let's make these `pub` instead

Reply:
Fine, just did it.